packages/blocks/src/store/private-selectors.js

```diff
@@ -16,6 +16,8 @@ const ROOT_BLOCK_SUPPORTS = [
 	'color',
 	'linkColor',
 	'buttonColor',
+	'headingColor',
+	'headingBackgroundColor',
```

Comment:
For the button and link colors, we don't separate between, text and background colors, should we follow the same approach here for simplicity?